customizations/mei-Mensural.xml

```diff
@@ -105,6 +105,34 @@
         <classSpec ident="model.sectionPart.cmn" module="MEI.shared" type="model" mode="delete"/>
         <classSpec ident="model.layerPart.cmn" module="MEI.shared" type="model" mode="delete"/>
 
+        <!-- Change the availability of dots -->
+        <!-- Remove them as children of <note> -->
+        <!-- Allow them as children of <ligature> -->
+        <elementSpec ident="dot" module="MEI.shared" mode="change">
+          <desc>Dot of augmentation or division.</desc>
+          <classes>
+            <memberOf key="att.common"/>
+            <memberOf key="att.facsimile"/>
+            <memberOf key="att.dot.log"/>
+            <memberOf key="att.dot.vis"/>
+            <memberOf key="att.dot.ges"/>
+            <memberOf key="att.dot.anl"/>
+            <memberOf key="model.noteModifierLike" mode="delete"/>
+            <memberOf key="model.eventLike.mensural" mode="add"/>
+          </classes>
+          <content>
+            <rng:empty/>
+          </content>
+          <remarks>
+            <p>This element provides an alternative to the <att>dots</att> attribute on <gi scheme="MEI"
```

Comment:
You should probably change this description to reflect that this element cannot be used to encode a dot of augmentation any more.

Reply:
But yes, you would use the `<dot>` element for encoding both dots of division and augmentation (using `@form` with the values `"div"` or `"aug"`, respectively). What you would not use anymore is the `@dots` attribute included in that description. 

So, I think that the description is still applicable. Isn't it?

Reply:
But yes, you would still use the `<dot>` element for encoding both dots of division and augmentation (using `@form` with the values `"div"` or `"aug"`, respectively). What you would not use anymore is the `@dots` attribute included in that description. 

What is changed by these few committed lines would be the place where this `<dot>` element is used—allowing it as a child of `<ligature>` and forbidding it as a child of `<note>`—not the features encoded by it.

So, I think that the description is still applicable. Isn't it?